src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -26,6 +29,14 @@ function LineVis(): JSX.Element {
       >
         {/* Provide context again - https://github.com/react-spring/react-three-fiber/issues/262 */}
         <LineProvider {...props}>
+          <TooltipMesh
+            formatIndex={([x]) => `x=${Math.floor(x)}`}
+            formatValue={([x]) =>
+              data[Math.floor(x)]
+                ? format('.3f')(data[Math.floor(x)])
+                : undefined
```

Comment:
If the abscissa `x` does not correspond to a data point, `data[Math.floor(x)]` is NaN so we deactivate the tooltip by returning `undefined`.